tooling/eslint-plugin/package.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "@orbit-ui/eslint-plugin",
+    "author": "Workleap",
+    "description": "Workleap recommended ESLint rules and configurations when using Orbit.",
+    "version": "0.0.1",
+    "keywords": [
+        "workleap",
+        "eslint",
+        "eslintconfig",
+        "eslintplugin",
+        "eslint-config",
+        "eslint-plugin"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gsoft-inc/sg-orbit.git",
+        "directory": "tooling/eslint-plugin"
+    },
+    "license": "Apache-2.0",
+    "exports": {
+        ".": {
+            "require": "./dist/index.js",
+            "import": "./dist/index.js",
+            "types": "./dist/index.d.ts"
+        }
+    },
+    "types": "./dist/index.d.ts",
+    "files": [
+        "docs",
+        "dist",
+        "CHANGELOG.md",
+        "README.md"
+    ],
+    "scripts": {
+        "build": "tsup"
+    },
+    "dependencies": {
+        "eslint-plugin-jsx-a11y": "^6.7.1"
+    },
+    "peerDependencies": {
+        "eslint": "*"
+    },
+    "peerDependenciesMeta": {
+        "eslint": {
+            "optional": true
```

Comment:
I am not sure why `eslint` would be an optional dependency?

I notice that our web-configs projet also mark the `eslint` dependency as "optional". This is so annoying that package files doesn't allow comments!!!